api/base/views.py

```diff
@@ -6,57 +6,214 @@
+    identifier, and every registration (a permanent, time-stamped version of your projects and files) can be assigned a
+    DOI/ARK. You can use the OSF to measure your impact by monitoring the traffic to projects and files you make
+    public. With the OSF you have full control of what parts of your research are public and what remains private.
+
+    Beta notice: This API is currently a beta service.  You are encouraged to use the API and will receive support
+    when doing so, however, while the API remains in beta status, it may change without notice as a result of
+    product updates. The temporary beta status of the API will remain in place while it matures. In a future
+    release, the beta status will be removed, at which point we will provide details on how long we will support
+    the API V2 and under what circumstances it might change.
+
+    #General API Usage
+
+    The OSF API generally conforms to the [JSON-API v1.0 spec](http://jsonapi.org/format/1.0/).  Where exceptions
+    exists, they will be noted.  Each endpoint will have its own documentation, but there are some general principles.
+
+    ##Requests
+
+    ###Canonical URLs
+
+    All canonical URLs have trailing slashes.  A request to an endpoint without a trailing slash will result in a
+    301 redirect to the canonical URL.
+
+    ###Plurals
+
+    Endpoints are always pluralized.  `/users/`, not `/user/`, `/nodes/`, not `/node/`.
+
+    ###Common Actions
+
+    Every endpoint in the OSF API responds to `GET`, `HEAD`, and `OPTION` requests.  You must have adequate permissions
+    to interact with the endpoint.  Unauthorized use will result in 401 Unauthorized or 403 Forbidden responses.  Use
+    `HEAD` to probe an endpoint and make sure your headers are well-formed.  `GET` will return a representation of the
+    entity or entity collection referenced by the endpoint.  An `OPTIONS` request will return a JSON object that describes the
+    endpoint, including the name, a description, the acceptable request formats, the allowed response formats, and any
+    actions available via the endpoint.
+
+    ###Filtering
+
+    Entity collections can be filtered by adding a query parameter in the form:
+
+        filter[<fieldname>]=<matching information>
+
+    String queries are filtered using substring matching. For example, if you were trying to find [Lise
+    Meitner](http://en.wikipedia.org/wiki/Lise_Meitner):
+
+        /users/?filter[full_name]=meitn
+
+    You can filter on multiple fields, or the same field in different ways, by &-ing the query parameters together.
+
+        /users/?filter[full_name]=lise&filter[family_name]=mei
+
+    Boolean fields should be queried with `true` or `false`.
+
+        /nodes/?filter[registered]=true
+
+    ###Pagination
+
+    All entity collection endpoints respond to the `page` query parameter behavior as described in the [JSON-API pagination
+    spec](http://jsonapi.org/format/1.0/#crud).
```

Comment:
That being said, we provide pagination links (also according to the spec) and you are encouraged to use links rather than adding query parameters by hand.